recipe/meta.yaml

```
@@ -18,9 +18,10 @@ source:
     # - patches/amd-roc-2.7.0.diff
     - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
     - patches/0002-restore-macos-10.9-support.patch
+    - patches/0003-fix-lljit-symbols.patch
```

always good to add context
```suggestion
    # backports patch for issues with LLJIT, see
    # https://github.com/llvm/llvm-project/commit/122ebe3b500190b1f408e2e6db753853e297ba28
    - patches/0003-fix-lljit-symbols.patch
```